server/xpub-server/entities/manuscript/resolvers/updateManuscript.js

```diff
@@ -1,10 +1,33 @@
+const Joi = require('joi')
 const pug = require('pug')
+const dns = require('dns')
+const util = require('util')
 const config = require('config')
 const mailer = require('@pubsweet/component-send-email')
 const logger = require('@pubsweet/logger')
-
 const { Manuscript, User } = require('@elifesciences/xpub-model')
 
+const resolveMx = util.promisify(dns.resolveMx)
+
+const isValidEmail = async email => {
```

Comment:
We can move this function to the controller to start with, and stub it out for testing purposes.

Reply:
Yup - sorted in the new commit.